src/App.js

```diff
@@ -1,29 +1,17 @@
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
+import './componentsStyles/calculatorStyles.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <Calculator />
+      </div>
+    );
+  }
 }
 
 export default App;
```

Comment:
- [ ] Make sure you are having a class-based component instead of a single function in App.js.

Reply:
Please, reviewer, I don't seem to understand what this particular review you ask me to make, because the App.js file uses a class-based component and it is obvious here that the file was changed, having removed the single function and replaced it with a class-based component.